libraries/AP_Vehicle/AP_FixedWing.h

```diff
@@ -56,5 +56,6 @@ struct AP_FixedWing {
     // Bitfields of TKOFF_OPTIONS
     enum class TakeoffOption {
         THROTTLE_RANGE          = (1U << 0), // Unset: Max throttle. Set: Throttle range.
+        AUTOLAND_DIR_ON_ARM     = (1U << 1), // set dir for autoland on arm if compass in use.
```

Comment:
If this is *really* only for autoland then it should be a parameter in the autoland mode.

Capturing the takeoff direction could be useful elsewhere, however, so I'm not that fussed.